dd-sdk-android-realm/README.md

```diff
@@ -0,0 +1,58 @@
+# Datadog Integration for Realm
+
+## Getting Started 
+
+To include the Datadog integration for [Realm][1] in your project, simply add the
+following to your application's `build.gradle` file.
+
+```
+repositories {
+    maven { url "https://dl.bintray.com/datadog/datadog-maven" }
+}
+
+dependencies {
+    implementation "com.datadoghq:dd-sdk-android:<latest-version>"
+    implementation "com.datadoghq:dd-sdk-android-rx:<latest-version>"
```

Comment:
Shouldn't this be `-realm` instead of `rx`?

Reply:
Yeah...we don't need this dependency actually, good spot.